packages/core/src/components/button/_button.scss

```diff
@@ -260,6 +261,17 @@ a.pt-button {
   @extend .pt-button:disabled;
 }
 
+.pt-button-loading-spinner {
```

Comment:
Wait, nvm, I misread that. If this corresponds to the spinner container element, go for `.pt-button-spinner-container`